src/apps/watchfaces/watchface_fitness_tracking.cpp

```diff
@@ -0,0 +1,186 @@
+#include "./apps/watchfaces/watchface_fitness_tracking.h"
```

Comment:
I won't review this file in detail, as many issues where mentioned in its copy-pasted brother :grinning:

Reply:
I'll fix it soon..